Makefile

```diff
@@ -16,6 +17,13 @@ compile: deps
 test: compile
 	npm run test
 	
+test-all: compile
```

Comment:
Could we switch it so make test runs all tests and something like make test-framework runs base tests?

Reply:
Wouldnt that run all tests when each collector runs `make test`? Since their make file is derived from the base make file?

Reply:
I'd prefer `make test` to run paws lib tests only. And `make test-collectors` to run all collectors tests